src/RadialGauge/RadialGauge.story.tsx

```diff
@@ -126,6 +129,119 @@ export const Stacked = () => {
   );
 };
 
+interface GaugeStackedTemplateProps {
+  width: number;
+  height: number;
+  data: ChartDataShape[];
+  maxValue?: number;
+  colorSchemaType: 'handlerFn' | 'array' | 'string';
+}
+
+const GaugeStackedTemplate: StoryFn<GaugeStackedTemplateProps> = (args) => {
```

Comment:
prefer `props` instead of `args` just for consistency but even better, prefer destructuring